tests/drivers/uart/uart_basic_api/testcase.yaml

```diff
@@ -7,3 +7,7 @@ tests:
       extra_args: CONF_FILE=prj_shell.conf
       platform_whitelist: quark_se_c1000_devboard quark_d2000_crb
       tags: drivers
+  - test_uart_poll:
+      extra_args: CONF_FILE=prj_poll.conf
+      platform_whitelist: esp32
```

Comment:
this platform is not supported in the SDK, so this testcase will never build in CI now, please set something else or have it build on qemu, there is nothing ESP32 specific about this test.
Also, the test is interactive, so it would fail in CI if it was run, this should be build_only in this case